apps/qubit/templates/layout_3col.php

```diff
@@ -22,6 +22,8 @@
 
     <div id="wrapper" class="container">
 
+      <?php echo get_partial('alerts') ?>
```

Comment:
I think we should add it too in the layout_wide, used in the DO browse and the IO inventory pages